.gitlint

```diff
@@ -0,0 +1,11 @@
+[title-max-length] 
+line-length=50
+
```

Comment:
Initial suggestion, I'll test the regex if it works unescaped, might need some work still...
```suggestion
[ignore-body-lines]
# Ignore all lines containing >72 character http:// or https:// URLs
# because URLs can not be easily wrapped to multiple lines without
# breaking them somehow.
#
# Explanation of the regular expression:
#
# "(?=\S{73,})" uses a positive lookahead with 73 non-whitespace characters
# which means there must be at least 73 non-whitespace characters ahead.
# It doesn't change the position in input string though. So if there are at
# least 73 non-whitespace characters ahead then those must start with
# either "http://" or "https://".
regex=(?=\S{73,})(http://|https://)
```

Reply:
Put these changes into PR #424